pkg/ottl/ottlfuncs/func_format_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ottlfuncs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+	"github.com/stretchr/testify/assert"
+)
+
+type getterFunc[K any] func(ctx context.Context, tCtx K) (any, error)
+
+func (g getterFunc[K]) Get(ctx context.Context, tCtx K) (any, error) {
+	return g(ctx, tCtx)
+}
+
+func Test_Format(t *testing.T) {
```

Comment:
Another test that confirms the behavior when a Getter returns an error would be nice.

Reply:
Done